.golangci.yml

```diff
@@ -2,6 +2,11 @@ run:
   tests: false
 
 linters-settings:
+  revive:
```

Comment:
I look around and there are many examples even in our codebase that follows the rule of using underscore, i.e. https://github.com/linode/linodego/blob/cfc31f1d4d543943a8b880221304eaa64bd6cf5f/instances.go#L279 I'm not seeing any disadvantage of doing it.

Reply:
The difference is this was in function body, and the unused variable in a function should be mark as an underscore which is enforced by the go lang compiler. The linter enforce it for the arguments list. It requires some changes like this:
<img width="735" alt="image" src="https://user-images.githubusercontent.com/121905282/233719113-75bf33cf-52d3-42cf-a01b-fc107524044a.png">

I wouldn't oppose to apply this linter, and I just committed [the fix](https://github.com/linode/linodego/pull/320/commits/2d9cd83e40450f7c52a6c4e87dfd589aa54a0408).

Reply:
Ideally we should remove this parameter but it will be a breaking change if we do so.